Update ESPHome 2026.2.4

Hello,

since ESPHome V2026.2.x i have an update error.
Does anyone have any ideas?

Here's the log:


*****************************************************************************
* Looking for NetworkClient.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:NetworkClient.h"
* Web  > https://registry.platformio.org/search?q=header:%1B%5Bm%1B%5BKNetworkClient.h
*
*****************************************************************************

   36 | #include <NetworkClient.h>
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/nspanel/src/esphome/components/nspanel_lovelace/nspanel_lovelace_upload.cpp.o] Error 1
In file included from src/esphome/components/nspanel_lovelace/nspanel_lovelace.h:13,
                 from src/esphome/components/nspanel_lovelace/nspanel_lovelace.cpp:2:
/data/cache/platformio/packages/framework-arduinoespressif32/libraries/HTTPClient/src/HTTPClient.h:36:10: fatal error: NetworkClient.h: No such file or directory

*****************************************************************************
* Looking for NetworkClient.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:NetworkClient.h"
* Web  > https://registry.platformio.org/search?q=header:%1B%5Bm%1B%5BKNetworkClient.h
*
*****************************************************************************

   36 | #include <NetworkClient.h>
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/nspanel/src/esphome/components/nspanel_lovelace/nspanel_lovelace.cpp.o] Error 1
========================= [FAILED] Took 30.19 seconds =========================

I have the same issue, and I (think?) I am pointing to dev already.

ESPHome 2026.2.4

and

```
external_components:
  - source:
      type: git
      url: https://github.com/sairon/esphome-nspanel-lovelace-ui
      ref: dev
    components: [nspanel_lovelace]
```


After a clean build:

```
Dependency Graph
|-- noise-c @ 0.1.10
|-- NetworkClientSecure @ 3.3.7
|-- HTTPClient @ 3.3.7

...
...

Compiling .pioenvs/sonoffnspanel1/src/esphome/components/nspanel_lovelace/nspanel_lovelace_upload.cpp.o
In file included from src/esphome/components/nspanel_lovelace/nspanel_lovelace.h:13,
                 from src/esphome/components/nspanel_lovelace/nspanel_lovelace.cpp:2:
/config/.esphome/platformio/packages/framework-arduinoespressif32/libraries/HTTPClient/src/HTTPClient.h:36:10: fatal error: NetworkClient.h: No such file or directory

*****************************************************************************
* Looking for NetworkClient.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:NetworkClient.h"
* Web  > https://registry.platformio.org/search?q=header:%1B%5Bm%1B%5BKNetworkClient.h
*
*****************************************************************************

   36 | #include <NetworkClient.h>
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/sonoffnspanel1/src/esphome/components/nspanel_lovelace/nspanel_lovelace.cpp.o] Error 1
In file included from src/esphome/components/nspanel_lovelace/nspanel_lovelace.h:13,
                 from src/esphome/components/nspanel_lovelace/nspanel_lovelace_upload.cpp:1:
/config/.esphome/platformio/packages/framework-arduinoespressif32/libraries/HTTPClient/src/HTTPClient.h:36:10: fatal error: NetworkClient.h: No such file or directory

*****************************************************************************
* Looking for NetworkClient.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:NetworkClient.h"
* Web  > https://registry.platformio.org/search?q=header:%1B%5Bm%1B%5BKNetworkClient.h
*
*****************************************************************************

   36 | #include <NetworkClient.h>
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/sonoffnspanel1/src/esphome/components/nspanel_lovelace/nspanel_lovelace_upload.cpp.o] Error 1
```

Hope this helps !!!

Thank you very much !

--- a/components/nspanel_lovelace/__init__.py
+++ b/components/nspanel_lovelace/__init__.py
@@ -83,6 +83,7 @@ async def to_code(config):
         await automation.build_automation(trigger, [(cg.std_string, "x")], conf)
 
     if CORE.is_esp32 and CORE.using_arduino:
+        cg.add_library("Networking", None)
         cg.add_library("NetworkClientSecure", None)
         cg.add_library("HTTPClient", None)
     elif CORE.is_esp32:

